.gitignore

```diff
@@ -1,3 +1,4 @@
 zig-cache/
 build/
 build-*/
+.vscode/
```

Comment:
See #888

Reply:
Fair enough I'll remove it :), I shouldn't be calling 'git add .' anyways.

Reply:
I think you should be able to use `git add .`. Everything to ignore falls into 2 categories: system-wide ignore, or project-specific ignore. If both are correct then `git add .` works just fine.